src/components/Stepper.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
```

Comment:
Same here. Make it a general component with variables

Reply:
Done it has been converted to a general component